test/functionalTest/cases/3122_location_metadata_issues/create_location_attr_V2.test

```diff
@@ -0,0 +1,166 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Create 2 entities with NGSIV2 POST.
+One with the location metadata, managed as a regular "custom" metadata as it is NGSIV2.
+The other without a location metadata but with a geo:point attribute, then managed as
+a location attribute for NGSIV2.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create E1 with NGSIV2
+# 02. Create E2 with NGSIV2
+# 03. Check E1 entity in DB
+# 04. Check E2 entity in DB
+
```

Comment:
I would suggest to use more precise steps (including echo messages) in this test. For instance

```
# 01. Create E1 with NGSIV2 with location metadata but not geo:xxx type
# 02. Create E2 with NGSIV2 without location metadata but with geo:xxx type
# 03. Check E1 entity in DB has location metadata but no location GeoJSON field
# 04. Check E2 entity in DB has no location metadata but location GeoJSON field
```

Reply:
Fixed in c76622d